### PR TITLE
822 add details to event logs

### PR DESCRIPTION
**Pull request by ywkim312**
## Description
Add more information of the events like tags, title, and description when POST, PUT, DELTE, and PATCH.

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [x] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: eventservice/api/utils/jsonutils.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+from flask import make_response
+
+
+def create_log_json(ep_name, ep_method, in_json):
+    """
+    create a json for logging
+    """
+    out_json = {}
+    out_json['ep_building_block'] = "event_service_building_block"
```
Comment (sandeep-ps) [inline eventservice/api/utils/jsonutils.py:23]:
```suggestion
    out_json['ep_building_block'] = "events_building_block"
```

File: eventservice/api/utils/jsonutils.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    out_json['ep_name'] = ep_name
+    out_json['ep_method'] = ep_method
+    if 'tags' in in_json:
+        out_json['tags'] = in_json['tags']
+    if 'title' in in_json:
+        out_json['title'] = in_json['title']
+    if 'description' in in_json:
```
Comment (sandeep-ps) [inline eventservice/api/utils/jsonutils.py:30]: The field name is `longDecription`. It can be stored as `longDescription` in the dictionary.

**Comment by sandeep-ps** [comment]
@ywkim312, I have left a couple of comments. The rest looks good to me. Thank you!

File: eventservice/api/utils/jsonutils.py (hunk 2 above)
Comment (ywkim312) [reply eventservice/api/utils/jsonutils.py:30]: I don't see longDescription in the README's example but description. Are you sure if it is longDescription?

File: eventservice/api/utils/jsonutils.py (hunk 1 above)
Comment (ywkim312) [reply eventservice/api/utils/jsonutils.py]: changed.

File: eventservice/api/utils/jsonutils.py (hunk 2 above)
Comment (sandeep-ps) [reply eventservice/api/utils/jsonutils.py:30]: As per the API schema, it's longDescription. Let me what's being used by the app.

Comment (sandeep-ps) [reply eventservice/api/utils/jsonutils.py:30]: It looks like the app may have used "description" at some stage. 
I think we can do something like this to account for that.

# Adding for potential backward compatibility.
if 'description' in in_json:
    out_json['longDescription'] = in_json['description']
elif 'longDescription' in in_json:
    out_json['longDescription'] = in_json['longDescription']

Comment (ywkim312) [reply eventservice/api/utils/jsonutils.py:30]: Updated for both code and README

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Added
```
Comment (sandeep-ps) [inline CHANGELOG.md:8]: Suggesting version number for this release.
 
```suggestion

## [1.11.3] - 2021-11-18
### Added
```

Comment (ywkim312) [reply CHANGELOG.md:8]: changed

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**